table-expressions.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Table Expressions
+summary: Table expressions define a data source in SELECT clauses and INSERT statements.
+toc: false
+---
+
+Table expressions define a data source in SELECT clauses and INSERT statements.
+
+<div id="toc"></div>
+
+## Introduction
+
+Table expressions are used prominently in the `SELECT` clause:
+
+	SELECT ... FROM <table expr>, <table expr>, ...
+
+CockroachDB recognizes the following table expressions:
+
+- a table or view name
+- a table generator function
+- a `SELECT` or `VALUES` clause, as a sub-query
+- an aliased table expression, using an `AS` clause
+- an explicit `JOIN` expression
+- another table expression annoted with `WITH ORDINALITY`
+- another table expression between parentheses,
+
+The following sections provide details on each of these options.
+
+In addition to this, the `FROM` clause itself accepts more than one
+consecutive table expressions at the top level, separated by
+commas. This is a shorthand notation for `CROSS JOIN`, documented in
+the `JOIN` syntax below.
+
+## Table or view names
+
+Syntax:
+
+	 identifier
+	 identifier.identifier
+
+A single SQL identifier in a table expression context designates
+the contents of the table or view with that name
+in the current database, as configured by [`SET DATABASE`](set-database.html).
+
+If the name is prefixed by another identifier and a period, the table or view
+is searched in the database with that name.
+
+For example:
+
+	SELECT * FROM users -- uses table `users` in the current database
+	SELECT * FROM mydb.users -- uses table `users` in database `mydb`
+
+## Table generator functions
+
+Syntax:
+
+	 name ( arguments... )
+
+(The name of a table generator function, followed by an opening
```

Comment:
Why is this in parentheses? Also, I can't quite make sense of this section in the context of this page. Can you add more context and an example?